accepted/00000-trust-vendor-gpg-keys.md

```diff
@@ -0,0 +1,75 @@
+- Feature Name: trust-vendor-gpg-keys
+- Start Date: 2020-03-25
+- RFC PR: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Provide a secure way to trust GPG keys needed for reposyncing and later to install packages on Clients.
+
+# Motivation
+[motivation]: #motivation
+
+When connecting Uyuni to SCC we provide a Product page where products provided by SUSE can be synced.
+But we provide also 3rd party repositories where metadata or packages are singed with not SUSE GPG keys.
+
+We need a way to provide the key and the info which key belong to which repository.
+We need a secure way to ask the admin if he wants to trust the key.
+
+After we got the permissions from the admin we can import the key in the keyring and also deploy the key
+to clients when they use the channels.
+
+# Detailed design
+[design]: #detailed-design
+
+## How to provide the keys?
+
+We should build a package which contains all the keys and get installed as files at `/usr/share/susemanager/gpg-keys/<keyid>.key` ?
```

Comment:
Is this it? `distribution-gpg-keys`

https://software.opensuse.org/package/distribution-gpg-keys

Reply:
Well, this package has a lot of keys, but all in different formats and sometimes multiple in 1 file. So it seems to be better to create a new package with content we can really use.

Reply:
I know this is many months later, but if you have a problem with how it's structured, talk to @xsuchy (upstream developer) and me (openSUSE maintainer) about it. Please don't make another random package for this because it just makes life miserable for everyone.

Reply:
*nod* I have no problem modifying the structure of distribution-gpg-keys. Just state your problem and suggested solution - ideally on https://github.com/xsuchy/distribution-gpg-keys/issues - and I will be happy to work with you on the solution.